build/tsconfig.common.json

```diff
@@ -6,7 +6,7 @@
     "baseUrl": "../",
     "paths": {
       "@stardust-ui/react": ["packages/react/src"],
-      "*": ["*", "types/*"]
+      "*": ["packages/react/*", "*", "types/*"]
```

Comment:
lets prioritise 'one-to-one' path mapping option (`*`) as the first thing to try - this will ensure correctness in cases of names collision

Reply:
Want to mention: I'm happy that we have there a star at all.